src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let finalPrice = 0;
+  const price = 40 * days;
+  const longVacation = 7;
```

Comment:
The variable name `longVacation` could be more descriptive. Consider renaming it to `LONG_TERM` to better reflect its purpose as a constant for the number of days required for a long-term discount.